apps/widget/cypress/e2e/initialization.spec.ts

```diff
@@ -17,8 +17,12 @@ describe('Initialization', function () {
 describe('Initialization with enabled HMAC encryption', function () {
   it('should initialize encrypted session with the help of HMAC hash', function () {
     cy.intercept('**/widgets/session/initialize**').as('sessionInitialize');
-    cy.initializeSession({ hmacEncryption: true });
-    cy.wait('@sessionInitialize');
+    cy.initializeSession({ hmacEncryption: true }).then(() => {
+      cy.wait(500);
+    });
+    cy.wait('@sessionInitialize', {
+      timeout: 60000,
+    });
```

Comment:
This test was flakey about 50% of the time, adding a wait eliminated flakiness in my local testing.